src/Functions/URL/cutURLParameter.cpp

```diff
@@ -1,82 +1,183 @@
+#include <Columns/ColumnArray.h>
+#include <Columns/ColumnConst.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnFixedString.h>
+#include <Columns/ColumnVector.h>
+#include <DataTypes/DataTypeArray.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
```

Comment:
Not needed

Reply:
Tried to remove these lines but it does not compile. Ho do you know which headers are not needed? Tidy? 
So I will try again to remove the unneeded headers tomorrow.

Reply:
By educated guesses. But if it doesn't compile w/o these headers, then please retain them, of course.

Reply:
OK. Thank you for clarifying. I don't use tidy (probably should turn it on) so I thought that some kind of automated tool like IntelliSense or tidy highlighted the unused headers. OK. I will manually check each header and remove unneeded ones.

Reply:
Done. Finally removed unneeded headers.